internals/secrethub/run.go

```diff
@@ -137,21 +137,14 @@ func (cmd *RunCommand) Run() error {
 		}
 	}
 
-	var parser tpl.Parser
-	switch cmd.templateVersion {
-	case "auto":
-		// Leave it up to the env file parser to detect the template version to use;
-		// The latest version is used for .env formatted environment files, v1 is used
-		// for yml formatted environment files.
-		parser = nil
-	case "1", "v1":
-		parser = tpl.NewV1Parser()
-	case "2", "v2":
-		parser = tpl.NewV2Parser()
-	case "latest":
-		parser = tpl.NewParser()
-	default:
-		return ErrUnknownTemplateVersion(cmd.templateVersion)
+	raw, err := ioutil.ReadFile(cmd.envFile)
```

Comment:
Why are we reading a file now? What changed that we need this here?

Reply:
It seems to me you're always reading the `.env` file. What happens when there's no `secrethub.env` file and no `--env-file` flag specified? And if you add `--envar` to that situation?